feat: Move files into GateIN and social - Meeds-io/MIPs#52 - MEED-1891

boubaker opened the pull request:
This change will:
* Centralize addons containers definition in GateIN-Portal for better maintainability and evolutivity
* Move thrid party JS libraries definition into social for better maintainability and evolutivity
* Delete deprecated and useless polyfill files
* Delete useless gtmpl files for Meeds Package
* Move Deprecated VueChart.js into wallet which remains the only addon using it
* Delete useless WebUI artifacts definition from Commons project
* Delete deprecated and useless insertPortalLink CKEditor plugin
* Use same style as portal in CKEditor and adjust specific CKEditor Style

boubaker committed:
feat: Move files into GateIN and social - Meeds-io/MIPs#52 - MEED-1891

This change will:
* Centralize addons containers definition in GateIN-Portal for better
maintainability and evolutivity
* Move thrid party JS libraries definition into social for better
maintainability and evolutivity
* Delete deprecated and useless polyfill files
* Delete useless gtmpl files for Meeds Package
* Move Deprecated VueChart.js into wallet which remains the only addon
using it
* Delete useless WebUI artifacts definition from Commons project
* Delete deprecated and useless insertPortalLink CKEditor plugin
* Use same style as portal in CKEditor and adjust specific CKEditor
Style

boubaker committed:
fix: Composer Scrollbar display - MEED-1945 - Meeds-io/meeds#802 (#463)

rdenarie commented:
Do you still use this file : https://github.com/Meeds-io/commons/blob/feature/mips/commons-extension-webapp/src/main/webapp/eXoPlugins.js

boubaker commented:
> Do you still use this file : https://github.com/Meeds-io/commons/blob/feature/mips/commons-extension-webapp/src/main/webapp/eXoPlugins.js

Nope, but this is another technical item in the backlog in fact, to make CKEditor extensible and clean it by moving eXo items into their respective addons.

boubaker committed:
fix: Fix CKEditor dialog style - MEED-1955 - Meeds-io/MIPs#52 (#466)

Prior to this change, the dialogs in CKEditor was displayed in full
width of the page and the actions not centered. This change will
introduce CSS for CKEditor which was deprecated on platform-ui.
